params:
  # Training parameters
  max_steps:          1e6
  warmup_steps:       100
  ep_max_steps:       300
  eval_interval:      20000
  eval_episodes:      10
  layer_units:        [256, 256]
  learning_rate:      2e-4
  rb_size:            1e5
  rb_steps_per_iter:  1
  change_ep:          5
  starting_eps:       20
  alpha_learn_rate:   3e-4
  tau:                5e-3
  update_period:      1
  gamma:              0.99
  rew_scale_factor:   1

  policy:             'TD3'
  logdir:             '/root/dev_ws/src/Results/'
  model_dir:          "results/Test_retrain_24022023_103518"
  # evaluate:         # comment/uncomment to evaluate trained model. Mutually exclusive with retrain
  # retrain:          # comment/uncomment to retrain trained model. Mutually exclusive with evaluate

  # Robot parameters
  robot_name:         "jackal_ugv"
  max_lin_vel:        0.5
  min_lin_vel:        0.0
  max_ang_vel:        1.0
  min_ang_vel:        -1.0
  robot_type:         "rectangular" # rectangular or circular
  robot_radius:       0.3 # [m], If robot_type is cicrular
  robot_size:         [0.508, 0.430] # [m, m], Lenght, Width if robot_type is rectangular
  collision_tolerance: 0.08 # [m], tollerance of collision added to robot_radius or robot_size
  warning_tolerance:  0.1 # [m], warning window added to robot_radius or robot_size
  goal_tolerance:     0.40

  # World parameters
  world_name:         "new_indoor" #Leave empty to obtain no world
  data_name:          "new_indoor"

  # Task parameters
  task:               'goToPose' # goToPose, Following, Vineyards
  
  # Sensors parameters
  update_frequency:   20.0
  sensor:             'lidar' # lidar or rgb_camera or depth_camera
  visual_data:        'features'
  features:           12544
  channels:           1
  imu_enabled:        "true"
  camera_enabled:     "true"
  stereo_camera_enabled: "false"
  lidar_enabled:      "true"
  sensors_topic:
    rgb_topic:        "/camera/image_raw"
    depth_topic:      "/camera/depth/image_raw" #"/camera_color/depth_aligned_to_color" 
    laser_topic:      "/scan" #scan
    imu_topic:        "/imu/data" #imu_data
    odom_topic:       "/odom"
  laser_param:
    max_distance:     12.0 # Meters
    num_points:       36 # int divider of 360
  depth_param:
    dist_cutoff:      5 # Meters
    width:            112
    height:           112
    show_image:       False
  rgb_param:
    width:            640
    height:           480
    show_image:       False 